src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

is it a good idea to get an object from `Optional` if it could be `null`? Or, maybe, better to throw an exception if the object is `null`?